### DISCUSSION TITLE
Is `v-bind( ... )` supported within the styles of Vue SFCs?

### DISCUSSION THREAD

**Discussion by msimpson** [parcel-bundler/parcel#6617] [Q&A] [answered]
If I try something like:

```
<script>

  export default {

    data() {

      color: '#fff'

    }

  };

</script>

<style scoped>

  div {
  
    color: v-bind( color );
  
  }

<style>

<template>

  <div>Example</div>

</template>
```

I see an attempt is being made in the compiled styles, such as: `color: var(--0eb8a5-color_)`.

However, I cannot find where that variable is being set.

A search of the entire build directory returns only where it's used.

Seems like a bug, but I wanted to ask first.

I'm using Parcel `2.0.0-nightly.776`.

**Comment by msimpson** [comment, marked as answer]
Never mind. A notice regarding `v-bind()` appeared in the build console after a reinstall.

**Reply by othy54** [reply]
How do you solve it ? I have the same problem :/

**Reply by msimpson** [reply]
The notice I mentioned above was about Parcel's lack of support for `v-bind` in styles.

So, I merely worked around it, e.g.:

```
<script>

  export default {

    data() {

      return { width: 0 };

    }

  };

</script>

<template>

    <div :style="{ '--width': width }"></div>

</template>

<style scoped>

  div {

    width: calc( var( --width ) * 1px );

  }

</style>
```
